scripts/build_experimental_archive.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
```

Comment:
I suggest to include the following line right at the beginning:

```suggestion
#!/bin/sh
set -euxo pipefail
```

Here's what it does:

1) `-e` terminates the script at the first error

2) `-u` terminates the script if you use any undefined variable

3) `-x` prints each command that is executed (very useful for debugging)

4) `-o pipefail` causes a pipeline to fail if any command inside a pipeline fails